bciers/apps/administration/tests/components/operators/OperatorPage.test.tsx

```diff
@@ -52,6 +56,9 @@ describe("Operator component", () => {
   it("renders the operator form with form data", async () => {
     // Mock the session data
     mockUseSession();
+    auth.mockReturnValueOnce({
+      user: { app_role: "industry_user_admin" },
+    });
```

Comment:
I ~think I have to mock both of these because `auth` is used in the server component and `useSession` in the client component it renders. Suggestions for a cleaner way always welcome!